src/templates/tabs/settingsgateway.jade

```diff
@@ -6,26 +6,27 @@ section.col-xs-12.content(ng-controller="SettingsGatewayCtrl")
       .alert.alert-success(ng-show="success.advanced_feature_switch", l10n) Trustline advanced settings have been changed successfully.
-      .alert.alert-info(ng-show="edit.defaultRippleFlagSaving", l10n) Updating DefaultRipple flag...
-      .alert.alert-success(ng-show="notif == 'defaultRippleUpdated'", l10n) DefaultRipple flag updated
-      .alert.alert-info(ng-show="edit.requireAuthFlagSaving", l10n) Updating RequireAuth flag...
-      .alert.alert-success(ng-show="notif == 'requireAuthUpdated'", l10n) RequireAuth flag updated
-      .alert.alert-danger(ng-show="notif == 'requireAuthFailed'", l10n) RequireAuth flag update failed. {{engine_result}}: {{engine_result_message}}
-      .alert.alert-info(ng-show="edit.globalFreezeFlagSaving", l10n) Updating GlobalFreeze flag...
-      .alert.alert-success(ng-show="notif == 'globalFreezeUpdated'", l10n) GlobalFreeze flag updated
-      .alert.alert-danger(ng-show="notif == 'globalFreezeFailed'", l10n) GlobalFreeze flag update failed. {{engine_result}}: {{engine_result_message}}
+      .alert.alert-info(ng-show="edit.defaultRippleFlagSaving && onlineMode", l10n) Updating DefaultRipple flag...
+      .alert.alert-success(ng-show="notif == 'defaultRippleUpdated && onlineMode'", l10n) DefaultRipple flag updated
+      .alert.alert-info(ng-show="edit.requireAuthFlagSaving && onlineMode", l10n) Updating RequireAuth flag...
+      .alert.alert-success(ng-show="notif == 'requireAuthUpdated && onlineMode'", l10n) RequireAuth flag updated
+      .alert.alert-danger(ng-show="notif == 'requireAuthFailed && onlineMode'", l10n) RequireAuth flag update failed. {{engine_result}}: {{engine_result_message}}
+      .alert.alert-info(ng-show="edit.globalFreezeFlagSaving && onlineMode", l10n) Updating GlobalFreeze flag...
+      .alert.alert-success(ng-show="notif == 'globalFreezeUpdated && onlineMode'", l10n) GlobalFreeze flag updated
+      .alert.alert-danger(ng-show="notif == 'globalFreezeFailed && onlineMode'", l10n) GlobalFreeze flag update failed. {{engine_result}}: {{engine_result_message}}
     .col-sm-3
       include settings/_navbar
-    group.disconnected(ng-hide="connected")
-      p.literal(l10n) You have to be online to see this screen
 
-    .col-sm-9.list(ng-hide='account.Balance')
+    .col-sm-9.list(ng-show="!loadingAccount && !account.Balance && loadState.account && connected")
       .unfunded.literal(l10n)
         | Your account has to be&#32;
         a(href="https://ripple.com/knowledge_center/activating-your-wallet/", target="_blank", l10n-inc) activated
         | &#32;before you can see this page.
 
-    .col-sm-9.list(ng-show='connected && account.Balance')
+    .col-sm-9.list(ng-show='((connected && account.Balance) || !onlineMode)')
+      //- Offline Flag Change
+      div(ng-show='offlineSettingsChange')
+        signed-transaction(data="{{signedTransaction}}", sequence="{{send.sequence}}")
```

Comment:
Is `send.sequence` defined in settingsgateway.js?  Didn't see it, but I might have missed it.